File: Python/browser/set_user_agent.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    context = browser.new_context(user_agent="Your-User-Agent/1.0")
    page = context.new_page()
    page.goto("https://example.com/")
    print(page.content())
    browser.close()

File: NodeJS/browser/set_user_agent.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const context = await browser.newContext({
    userAgent: 'Your-User-Agent/1.0'
  });
  const page = await context.newPage();
  await page.goto('https://httpbin.org/headers');
  console.log(await page.content());
  await browser.close();
})();
